showcase/app/templates/components/advanced-table.hbs

```diff
@@ -112,6 +112,44 @@
 
   <Shw::Divider />
 
+  <Shw::Text::H2>Horizontal scrolling indicators</Shw::Text::H2>
+  <div class="shw-component-advanced-table-fixed-dimensions-wrapper">
+    <Hds::AdvancedTable
+      @model={{this.model.music}}
+      @isStriped={{true}}
+      @columns={{array
+        (hash key="artist" label="Artist" tooltip="More information." isSortable=true)
+        (hash key="album" label="Album" tooltip="More information." isSortable=true)
+        (hash key="year" label="Release Year" tooltip="More information." isSortable=true)
+        (hash key="other" label="Additional Actions")
+      }}
+    >
+      <:body as |B|>
+        <B.Tr>
+          <B.Th @scope="row"><Hds::Link::Inline @href="#showcase">{{B.data.artist}}</Hds::Link::Inline></B.Th>
+          <B.Td>
+            <div class="shw-component-advanced-table-cell-content-div">
+              <Hds::Icon @name={{B.data.icon}} @isInline={{true}} />
+              {{B.data.album}}
+            </div>
+          </B.Td>
+          <B.Td>
+            <Hds::Badge @text={{B.data.year}} @type={{B.data.badge-type}} @color={{B.data.badge-color.name}} />
+          </B.Td>
+          <B.Td>
+            <Hds::ButtonSet>
+              <Hds::Button @text="Add" @isIconOnly={{true}} @icon="plus" @size="small" />
+              <Hds::Button @text="Edit" @isIconOnly={{true}} @icon="edit" @size="small" @color="secondary" />
+              <Hds::Button @text="Delete" @isIconOnly={{true}} @icon="trash" @size="small" @color="critical" />
+            </Hds::ButtonSet>
+          </B.Td>
+        </B.Tr>
+      </:body>
+    </Hds::AdvancedTable>
+  </div>
+
+  <Shw::Divider />
+
   <Shw::Text::H2>Sticky header</Shw::Text::H2>
```

Comment:
<img width="599" alt="Screenshot 2025-03-24 at 1 48 46 PM" src="https://github.com/user-attachments/assets/c2283f01-6f38-443a-852c-c438b94d9b43" />

I can't quite remember if we came to a conclusion on this, but in a table with vertical scroll, I think I would expect to see the same type of shadow indicating scroll on the Y axis. Without this or without a border it looks like un-polished.

Reply:
Ah ok maybe we can talk about this -  I thought we only wanted to add a vertical scroll shadow below the sticky header.

Reply:
Bottom shadow has been added.